.gitignore

```diff
@@ -75,5 +75,5 @@ _build/
 stack*.yaml.lock
 shake.yaml.lock
 
-# ignore hie.yaml's for testdata 
-test/**/*.yaml
+# ignore hie.yaml
+hie.yaml
```

Comment:
Why is this here? Don't you need them to run tests?

Reply:
they are generated, just like the `stack.yaml`.